packages/ui/src/components/RowV2/index.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from '@emotion/styled'
+import type { CSSProperties, ForwardedRef, ReactNode } from 'react'
+import { forwardRef } from 'react'
+import type { SCWUITheme } from '../../theme'
+
+type StyledRowProps = Pick<RowV2Props, 'gap' | 'templateColumns' | 'alignItems'>
+export const StyledRow = styled('div', {
+  shouldForwardProp: prop =>
+    !['templateColumns', 'gap', 'alignItems'].includes(prop),
+})<StyledRowProps>`
+  display: grid;
+  ${({ theme, gap = 2, alignItems = 'normal', templateColumns }) => `
+    grid-template-columns: ${templateColumns};
+    grid-gap: ${theme.space[gap]};
+    align-items: ${alignItems};
+  `}
+`
+
+type RowV2Props = {
+  id?: string
+  className?: string
+  dataTestId?: string
+  children: ReactNode
+  templateColumns: string
+  gap?: keyof SCWUITheme['space']
+  alignItems?: CSSProperties['alignItems']
+}
+
+export const RowV2 = forwardRef(
```

Comment:
Why the forwardRef ? 🤔

Reply:
I think it's better to have it on every component (we think about that). It's optional but let say you wanna scroll to this row for some reason you can it's convenient and there are multiples use case like that.